node-export-server/server.js

```diff
@@ -275,23 +351,26 @@ function doExport(req, res, hash, input) {
         //handle different flavors of bbox param
         var bbox_param = 'convert.bounds';
         var bbox = exports.validateBbox(req.query.bbox);
-
+        var poly = exports.validatePoly(req.query.poly);
         if (input.substring(0,4) === 'http') {
             var cert_param = '';
             if (input.substring(0,5) === 'https' && fs.existsSync(appDir + 'key.pem') && fs.existsSync(appDir + 'cert.pem')) {
                 cert_param = '--private-key=' + appDir + 'key.pem --certificate=' + appDir + 'cert.pem';
             }
             temp_file = outDir + '.osm';
-            command += 'wget -q ' + cert_param + ' -O ' + temp_file + ' ' + input + '?bbox=' + bbox + ' && ';
+            command += 'wget -q ' + cert_param + ' -O ' + temp_file + ' ' + input + '?bbox=' + (poly ? exports.polyToBbox(poly) : bbox) + ' && ';
```

Comment:
If we ever added overpass as a node export datasource (https://github.com/ngageoint/hootenanny/issues/1474) we would be able to query with the poly directly instead of being limited by bbox of the OSM API.